pkg/nvlib/device/device.go

```diff
@@ -208,6 +209,33 @@ func (d *device) IsMigEnabled() (bool, error) {
 	return (mode == nvml.DEVICE_MIG_ENABLE), nil
 }
 
+// HasGpuFabricInfo() checks if a device has IMEX capabilities.
+func (d *device) HasGpuFabricInfo() (bool, error) {
+	if d.lib.hasSymbol("nvmlDeviceGetGpuFabricInfo") {
```

Comment:
Can we make this a:
```
if !d.lib.hasSymbol("nvmlDeviceGetGpuFabricInfo") {
	return false, nil
}
```

Reply:
the idea behind checking old API and then new API, is to flip it to call `nvmlDeviceGetGpuFabricInfo` if it exists and if not call `nvmlDeviceGetGpuFabricInfoV` if it exists.
if we return directly here, we could be in a newer system that already removed the `nvmlDeviceGetGpuFabricInfo` but may still have the `nvmlDeviceGetGpuFabricInfoV`

Reply:
Yeah, this was in relation to me saying we should drop `nvmlDeviceGetGpuFabricInfoV`. I am fine with keeping both though, so this comment is no longer relevant.